flask_hw/views.py

```diff
@@ -140,22 +151,35 @@ def login():
         if len(email) >= 5:
             validator = password_valid(password)
             if validator:
+                session['email'] = email
                 return redirect('/users')
             else:
                 abort(400, 'Invalid password')
         else:
             abort(400, 'Invalid data')
 
     else:
-        return f'''
-            <form method="POST" action="/login">
-                <label for="email">Email:</label>
-                <input type="email" name="email" id="email"><br><br>
-                <label for="password">Password:</label>
-                <input type="password" name="password" id="password"><br><br>
-                <input type="submit" value="Submit">
-            </form>
-            '''
+        return render_template('main/login.html'), 200
+
+
+@app.route('/logout')
+def logout():
+    session.pop('email', None)
+    return redirect('/login')
+
+
+def session_username(email):
```

Comment:
nit: could be implemented as a decorator to avoid repetition of code 'if session_username(email):'